misc/userscripts/qutebrowser_viewsource

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# 
+# This script fetches the unprocessed HTML source for a page and opens it in vim.
+#	:bind gf spawn --userscript qutebrowser_viewsource
+#
+
+path=/tmp/qutebrowser_$(mktemp XXXXXXXX).html
```

Comment:
I suggest using `path=$(mktemp -t qutebrowser_XXXXXXXX.html)` - that'll also respect `$TMPDIR` if the user set that.